yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
we don't need this file, the proper yarn.lock will be at hapi and webapp folder

Reply:
ok @adriexnet, gracias por la observación.